.github/release.yml

```diff
@@ -14,3 +14,6 @@ changelog:
     - title: Internal Improvements
       labels:
```

Comment:
There are some PRs that are labeled with the memory label we should probably add it to the internal improvements

Reply:
I've added a memory category